packages/convex-helpers/server/zod4.ts

```diff
@@ -0,0 +1,642 @@
+import { z } from "zod";
```

Comment:
```suggestion
import { z } from "zod/v4/core";
```

I think that this is change is necessary. With the current import, users of convex-helpers that use Zod 3 would get the `/zod4` types incorrectly defined based on Zod 3 (see https://zod.dev/library-authors?id=how-to-support-zod-4). According to the same page, we need to import from `zod/v4/core`, unless we do something that can’t work with Zod Mini (but they strongly recommend against doing so).